### ISSUE TITLE
Issue with starting with Windows

### ISSUE THREAD

**Issue by vertigo220** [ykon/w10wheel.net#5]
As discussed in issue #4 and added to the wiki, I added W10Wheel to the registry to run with Windows, but I rarely reboot my computer and when I finally did, I found it wasn't starting. It didn't matter whether it was in HKCU or HKLM, it wasn't working. I also tried adding it to the startup folder, both for all users and just me, and neither of those worked, either. I've never run into this issue before. It works fine if I run it after, either manually or by using another program, but not when using the registry or startup folders. It's like it won't run until a Windows service or something is running, at least that's my best guess. So it complicates things as far as getting it to run with Windows, whether by my instructions given before or possibly even by you building it in to the app.

**Comment by ykon** [comment]
Thanks again.

I'm sorry, but I haven't been able to register the registry yet because it's difficult to figure out what to do with the UI.
And unfortunately, I'm using the Task Scheduler instead of the registry. If you use the Task Scheduler, you can set a delay so that it starts after a 30-second delay, which I think will work.

Currently, I don't know why it doesn't start in the registry.

**Comment by vertigo220** [comment]
Remember, not just the registry. It doesn't start if you put it in the startup folder, either. Since you have it set up with task scheduler, it would be interesting to see if it runs at startup for you or if you _need_ to set a delay for it to work, same as with startup and registry.

**Comment by ykon** [comment]
Yes, it is. I'd like to look into it in more detail when I have more time.

**Comment by vertigo220** [comment]
I switched to using Task Scheduler to run this at Windows a while back, though that's definitely not my preferred way of doing so, and it still requires a delay there. I can't even remember anymore if I tested to see how much of a delay is needed, but I set it to 30 seconds and it always started, so recently I set it to 25 seconds and haven't noticed any issues. I just dropped it to 20 to see if that works, but the bottom line is it's a problem regardless of the method used.
